.travis.yml

```
@@ -8,7 +8,10 @@ cache:
 notifications:
   email: false
 node_js:
-  - 'lts/*'
+  - 'node'
```

```suggestion
  - '11'
```
there’s no need to have ci start failing randomly whenever node 12 comes out :-)

---

Why not? Who then keeps an eye on the CI matrix to bump it to 12? Also `node` is recommended and used in many projects. So even if you forget to add further LTS versions when a new stable branch is released, you always have the CI running against the current stable release which catches any new issues (many update early and 11 is a shortliving branch like every odd release branch).

If this is an issue for you, we can add `node` to `allow_failures`.

As most do not check the CI status, you will get at least a green status but would have to check the details page if it failed.

---

The same people who keep an eye on semver-major dependency updates, which don’t magically run on regular ci. Regular CI isn’t the place to just “hope” that after an update happens, someone notices a failure - that’s what things like greenkeeper are for.

Adding it to allowed failures should work fine tho, if you think this is valuable.

---

> Adding it to allowed failures should work fine tho, if you think this is valuable.

Ok, I'll add a commit tomorrow probably for `allow_failures`.